content/01.abstract.md

```diff
@@ -2,7 +2,7 @@
 
 The Single-cell Pediatric Cancer Atlas (ScPCA) Portal (<https://scpca.alexslemonade.org/>) is a data resource for uniformly processed single-cell and single-nuclei RNA sequencing (RNA-seq) data and de-identified metadata from pediatric tumor samples.
 Originally comprised of data from 10 projects funded by Alex’s Lemonade Stand Foundation (ALSF), the Portal currently contains summarized gene expression data for over 700 samples across 55 cancer types from ALSF-funded and community-contributed datasets.
-Downloads include gene expression data as `SingleCellExperiment` or `AnnData` objects containing raw and normalized counts, PCA and UMAP coordinates, and automated cell type annotations, along with summary reports.
+Downloads include gene expression data as `SingleCellExperiment` or `AnnData` objects containing raw and normalized counts, PCA and UMAP coordinates, automated cell type annotations, and summary reports.
```

Comment:
Okay so I had originally had the sentence like this but then it sounded like the objects contained the summary reports which is incorrect. If you think it's okay then I'm good with it, but just wanted to note that maybe the original sentence here is actually the correct meaning.

Reply:
done https://github.com/AlexsLemonade/ScPCA-manuscript/pull/173/commits/e3afdccc15bfa58a58a370f013c3cf02a865762c